JavaFiddle-ejb/src/main/java/ru/javafiddle/core/ejb/CompileAndRunBean.java

```diff
@@ -1,27 +1,200 @@
 package ru.javafiddle.core.ejb;
 
+import org.apache.log4j.BasicConfigurator;
+import org.apache.log4j.Logger;
+import ru.javafiddle.jpa.entity.File;
+import ru.ncedu.dynamic.ByteArrayResource;
+import ru.ncedu.dynamic.DynamicCompiler;
+import ru.ncedu.dynamic.StringJavaFileObject;
+
 import javax.ejb.Stateless;
 import javax.inject.Named;
 import javax.persistence.EntityManager;
 import javax.persistence.PersistenceContext;
+import javax.tools.SimpleJavaFileObject;
+import java.io.ByteArrayOutputStream;
+import java.io.PrintStream;
+import java.lang.reflect.InvocationTargetException;
+import java.lang.reflect.Method;
+import java.util.ArrayList;
+import java.util.List;
 
 /**
  * Created by artyom on 15.12.15.
  */
 @Stateless
 @Named(value = "compileAndRunBean")
-public class CompileAndRunBean {
+public class CompileAndRunBean extends DynamicCompiler {
 
     @PersistenceContext
     EntityManager em;
 
+
+    public static final String DEFAULT_PACKAGE_PREFFIX = "";
+    private static final Logger LOG = Logger.getLogger(CompileAndRunBean.class);
+
+    public void init(ClassLoader parentLoader) {
+        try {
+            super.init(parentLoader);
+        } catch (Exception e) {
+            LOG.error("Exception", e);
```

Comment:
What happens in case of an exception? As of now, init() method finishes successfully (with an error in logs).
Is the system able to continue operating?